libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -486,8 +486,7 @@ void AC_PosControl::relax_velocity_controller_xy()
     // decay resultant acceleration and therefore current attitude target to zero
     float decay = 1.0 - _dt / (_dt + POSCONTROL_RELAX_TC);
 
-    _accel_target.x *= decay;
-    _accel_target.y *= decay;
+    _accel_target.xy() *= decay;
```

Comment:
This apparently works but I must say the implementation in Vector3f seems very dodgy to me.